terracumber_config/tf_files/Uyuni-Master-refenv-NUE.tf

```diff
@@ -0,0 +1,194 @@
+// Mandatory variables for terracumber
+variable "URL_PREFIX" {
+  type = "string"
+  default = "https://ci.suse.de/view/Manager/view/Uyuni/job/uyuni-master-infra-reference-NUE"
+}
+
+// Not really used as this is for --runall parameter, and we run cucumber step by step
+variable "CUCUMBER_COMMAND" {
+  type = "string"
+  default = "export PRODUCT='Uyuni' && run-testsuite"
+}
+
+// Not really used in this pipeline, as we do not run cucumber
+variable "CUCUMBER_GITREPO" {
+  type = "string"
+  default = "https://github.com/uyuni-project/uyuni"
+}
+
+// Not really used in this pipeline, as we do not run cucumber
+variable "CUCUMBER_BRANCH" {
+  type = "string"
+  default = "master"
+}
+
+// Not really used in this pipeline, as we do not run cucumber
+variable "CUCUMBER_RESULTS" {
+  type = "string"
+  default = "/root/spacewalk/testsuite"
+}
+
+// Not really used in this pipeline, as we do not send emails on success (no cucumber results)
+variable "MAIL_SUBJECT" {
+  type = "string"
+  default = "Results Uyuni RefHead-NUE $status: $tests scenarios ($failures failed, $errors errors, $skipped skipped, $passed passed)"
+}
+
+variable "MAIL_TEMPLATE" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins.txt"
+}
+
+variable "MAIL_SUBJECT_ENV_FAIL" {
+  type = "string"
+  default = "Results Uyuni RefHead-NUE: Environment setup failed"
+}
+
+variable "MAIL_TEMPLATE_ENV_FAIL" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins-refenv-fail.txt"
+}
+
+variable "MAIL_FROM" {
+  type = "string"
+  default = "galaxy-ci@suse.de"
+}
+
+variable "MAIL_TO" {
+  type = "string"
+  default = "galaxy-ci@suse.de"
+}
+
+// sumaform specific variables
+variable "SCC_USER" {
+  type = "string"
+}
+
+variable "SCC_PASSWORD" {
+  type = "string"
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://cokerunner.mgr.suse.de/system"
+}
+
+module "base" {
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+
+  name_prefix = "uyuni-refhead-"
```

Comment:
Do you what? Now that we even merged the changes for DNS/DHCP, I'm not sure about this prefix.
Maybe instead of "head" it should contain the word "master".

@juliogonzalez what do you think, it's just fine with uyuni-refhead- as prefix?

Reply:
@srbarrios it should be `uyuni-refmaster` FMPOV. Changing DNS/DHCP should be really easy with `sed`, a PR, then applying the changes.

Reply:
Oki dokie, @nodeg sorry to don't catch it earlier!

Reply:
No problem. I will adjust it here and do a follow up PR in GitLab with the name changes, too.